src/v3/src/components/Title/Title.tsx

```diff
@@ -12,27 +12,40 @@
 
 import { Box, Typography } from '@okta/odyssey-react-mui';
 import { h } from 'preact';
+import { useEffect, useRef } from 'preact/hooks';
 
 import { TitleElement, UISchemaElementComponent } from '../../types';
 
 const Title: UISchemaElementComponent<{
   uischema: TitleElement
 }> = (
   { uischema: { id, options } },
-) => (
-  <Box
-    display="flex"
-    justifyContent="flex-start"
-  >
-    <Typography
-      id={id}
-      component="h2"
-      variant="h4"
-      data-se="o-form-head"
+) => {
+  const titleRef = useRef<HTMLTitleElement>(null);
+  useEffect(() => {
+    titleRef.current?.focus();
+  }, []);
```

Comment:
I am curious how does this work when autoFocus config option is set to true? Will it essentially be ignored? If so, we should probably check the autoFocus value to ensure this does not override that functionality? 

Also, is it semantically correct to focus on the Title instead of the H1? (Maybe that doesn't matter but just wanted to call it out to make sure we arent doing something unexpected).

Reply:
Agree we should probably check on the `autoFocus` option and perhaps ignore this effect if another element should be focused instead?

Reply:
autoFocus config option does still work properly when enabled, since the title gets focused first, and then the interactive element will take focus from it.  But I agree I think its better to add a check for `autoFocus` there to make sure.

Reply:
Yeah, good question! I think Title is best semantically. It's the start of the content that's changed, and its announcement will be what users want to know. (The H1 doesn't change between views.)